RecoMuon/GlobalTrackingTools/src/GlobalMuonRefitter.cc

```diff
@@ -118,8 +119,10 @@ GlobalMuonRefitter::GlobalMuonRefitter(const edm::ParameterSet& par,
 
   theDYTthrs     = par.getParameter< std::vector<int> >("DYTthrs");
   theDYTselector = par.existsAs<int>("DYTselector")?par.getParameter<int>("DYTselector"):1;
-  theDYTupdator = par.existsAs<bool>("DYTupdator")?par.getParameter<bool>("DYTupdator"):false;
-  theDYTuseAPE = par.existsAs<bool>("DYTuseAPE")?par.getParameter<bool>("DYTuseAPE"):false;
+  theDYTupdator  = par.existsAs<bool>("DYTupdator")?par.getParameter<bool>("DYTupdator"):false;
```

Comment:
I think you can define all these parameters with a ```fillDescriptions()``` method, and get rid of all these ```par.existsAs``` checks

Reply:
Dear @perrotta, could you please point me an example of how `fillDescriptions()` works?

Reply:
See https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp

You can find concrete examples in the vast majority of classes in CMSSW which have configurable parameters